dune

```diff
@@ -599,35 +601,85 @@
   (unbox_free_vars_of_closures.mli
    as
    compiler-libs/unbox_free_vars_of_closures.mli)
-  (.ocamloptcomp.objs/native/internal_assembler.cmx as compiler-libs/internal_assembler.cmx)
-  (.ocamloptcomp.objs/byte/internal_assembler.cmi as compiler-libs/internal_assembler.cmi)
-  (.ocamloptcomp.objs/byte/internal_assembler.cmo as compiler-libs/internal_assembler.cmo)
-  (.ocamloptcomp.objs/byte/internal_assembler.cmt as compiler-libs/internal_assembler.cmt)
-  (.ocamloptcomp.objs/byte/internal_assembler.cmti as compiler-libs/internal_assembler.cmti)
+  (.ocamloptcomp.objs/native/internal_assembler.cmx
```

Comment:
(Does not matter, but it looks it is not following the
convention: `cmx` files were in another list -- to be
clear the breakage of the convention was when
merging #757 I guess.)

Reply:
Discussed offline - I think this ok for the moment, but we should work out what's required in Dune (to enhance the `install` stanzas) so we can remove all of this.